### DISCUSSION TITLE
Any plans on adding chat function to streamlink (twitch)?

### DISCUSSION THREAD

**Discussion by Erik5000** [streamlink/streamlink#5562] [Q&A] [answered]
### Description

Hey :)

Would love to know if there is any plan to add a feature to chat with the live streams you watching? Would love to have access to the chat via CLI.
Or do you know any repo/project I can use for that?
I searched but couldn't find any good ones.

Cheers
Thank you

**Comment by bastimeyer** [comment, marked as answer]
> [Streamlink](https://streamlink.github.io/) is a command-line utility **which pipes video streams** from various services **into a video player**

Streamlink has literally nothing to do with anything non-streaming related, like chat, etc. This has also been answered multiple times, so if you search on the discussion forum or issue tracker, maybe even on the Streamlink Twitch GUI repo, you'll find more answers.

> Or do you know any repo/project I can use for that?

You can use an IRC client for Twitch chat. This is regular IRCv3. Twitch-chat-specific applications like Chatty, Chatterino, etc. however are not CLI/TUI applications but GUIs. Just use Google.

**Reply by Erik5000** [reply]
I tried to find anything that worked. Nothing worked. Sorry for asking.
